### PR TITLE
TXI Mirror Cooling, SP1K1 Mono RTDs

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- `XOffsetMirrorNoBend` in mirror.py gets 3 new cooling readout components.
- `Mono` in spectrometer.py gets 4 new RTD components and re-named RTDs 1-8. Make cooling component names consistent with mirror cooling component names.

## Motivation and Context

- Wired up cooling panel sensors.
- Found new information about the RTD positions on SP1K1 MONO.

## How Has This Been Tested?

- Ran typhos screens locally.

## Where Has This Been Documented?

https://jira.slac.stanford.edu/browse/ECS-4588

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Review by tangkong — COMMENTED** [review]
My guess is that nobody relies on these, but I'm always wary of changing existing names in case someone's script assumes it will be there.  Aside from that I'm in favor of the more descriptive names (rtd might be an understandable acronym to everyone)

**Comment by nrwslac** [comment]
> My guess is that nobody relies on these, but I'm always wary of changing existing names in case someone's script assumes it will be there. Aside from that I'm in favor of the more descriptive names (rtd might be an understandable acronym to everyone)

I could be convinced to drop rtd for temp.

**Comment by tangkong** [comment]
Now that you mention it, I'd be much happier with something like `grating_temp_1`.  At the point where someone's using an ophyd device I'm not sure the exact temperature measurement method/device is too important?

**Comment by nrwslac** [comment]
> Now that you mention it, I'd be much happier with something like `grating_temp_1`. At the point where someone's using an ophyd device I'm not sure the exact temperature measurement method/device is too important?

yea, I think temp is the way to go. I'll update.

File: pcdsdevices/mirror.py [hunk 1]
```diff
@@ -752,12 +752,16 @@ class XOffsetMirrorNoBend(XOffsetMirror):
 
     name : str
 
-    Currently (10/11/2023) services: mr1l1
+    Currently (10/11/2023) services: mr1l1, mr1k3, mr2k3
```
Comment (ZLLentz) [inline pcdsdevices/mirror.py:755]: Is the old date still correct in this comment?
```suggestion
    Currently (5/15/2024) services: mr1l1, mr1k3, mr2k3
```

**Comment by ZLLentz** [comment]
I'll drop my approval once you update the names

**Comment by nrwslac** [comment]
Updated mono temperature readout names and mirror doc string date.

**Review by ZLLentz — APPROVED** [review]
Looks good to me!

**Comment by ZLLentz** [comment]
I'm going to roll through with an admin merge due to the current docs build issues

**Pull request merged**